build.gradle

```diff
@@ -123,6 +123,7 @@ dependencies {
   compile "org.eclipse.jetty:jetty-server:9.4.8.v20171121"
   compile "org.eclipse.jetty:jetty-servlet:9.4.8.v20171121"
   compile "org.gitlab:java-gitlab-api:4.0.0"
+  compile "io.gitea:java-gitea-api:1.0.0"
```

Comment:
Hmm. Does such thing actually exist? Travis failed to download it.

Reply:
It's at  https://github.com/zeripath/java-gitea-api

Do I need to force publish it somewhere?

Reply:
Hmm. Ideally, it should just be published to Maven Central. Not sure what to do here. @bozaro any ideas?

Reply:
OK I've just changed the groupId and uploaded to sonatype - hopefully that will fix this.

Reply:
Oh, cool!